electrum/plugins/jade/jade.py

```diff
@@ -350,7 +350,7 @@ def show_address_multi(self, multisig_name, paths):
 class JadePlugin(HW_PluginBase):
     keystore_class = Jade_KeyStore
     minimum_library = (0, 0, 1)
-    DEVICE_IDS = [(0x10c4, 0xea60), (0x1a86, 0x55d4), (0x0403, 0x6001)]
+    DEVICE_IDS = [(0x10c4, 0xea60), (0x1a86, 0x55d4), (0x0403, 0x6001), (0x1a86, 0x7523)]
```

Comment:
Could you please add some newlines and comments to help distinguish which code is for what kind of device?
Something like this:
https://github.com/spesmilo/electrum/blob/827792c14cb11cfca7c8c91af2724c8b15fa2df4/electrum/plugins/ledger/ledger.py#L1323-L1335

Reply:
You are right, it certainly gets a bit unwieldly without that. Have added them in. :)